### DISCUSSION TITLE
Uninstall-Module error ; Access to the cloud file is denied

### DISCUSSION THREAD

**Discussion by francisco-lafe** [JanDeDobbeleer/oh-my-posh#2230] [Troubleshoot] [answered]
Hello, I've tried to follow the migration docs:
https://ohmyposh.dev/docs/migrating 
And when I issue this command `Uninstall-Module oh-my-posh -AllVersions`
I get this error:
```
PackageManagement\Uninstall-Package : Access to the cloud file is denied
At C:\Program Files\WindowsPowerShell\Modules\PowerShellGet\1.0.0.1\PSModule.psm1:2194 char:21
+ ...        $null = PackageManagement\Uninstall-Package @PSBoundParameters
```
![image](https://user-images.githubusercontent.com/90066435/167168740-674de9f0-9bc3-476a-bfaa-b790ee9b7b58.png)

I can't continue and I can't find an answer to this error. 
I thought it had to do with OneDrive but even with OneDrive disabled it still says it's unable to access. 
Then I thought maybe I need to have an elevated PS but still with no avail.

Help?
Thanks.

**Comment by JanDeDobbeleer** [comment, marked as answer]
No idea tbh, this is an issue in PowerShellGet and not oh-my-posh. Have a look [here](https://github.com/PowerShell/PowerShellGet/issues/262).

**Reply by francisco-lafe** [reply]
Ok, ty!
